magento/inventory
# Migrate in-store-pickup email to strict mode

### Description (*)

The In Store Pickup "Ready for pickup" notification email still used deprecated (and removed as of magento 2.4.4) variables with direct method calls.
This resulted in the affected variables in the email template not being rendered.

### Fixed Issues (if relevant)

1. Fixes magento/inventory#3384

### Manual testing scenarios (*)

1. Create order with in-store-pickup
2. Send ready for pickup email
3. Check email output for customer name.

### Questions or comments


### Contribution checklist (*)
